modules/cluster-autoscaler-about.adoc

```diff
@@ -58,6 +58,13 @@ The cluster autoscaler takes pod priorities into account. The Pod Priority and P
 
 Pods with priority lower than the cutoff value do not cause the cluster to scale up or prevent the cluster from scaling down. No new nodes are added to run the pods, and nodes running these pods might be deleted to free resources.
 
+The cluster autoscaling is supported on the following cloud providers:
```

Comment:
I am not sure if we really need this list,  as long as support machine-api, it also supports autoscaler, if we must list this, gcp, vsphere, ibmcloud, nutanix all these platforms support cluster autoscaler.

Reply:
i agree, we should specify that autoscaling works where the machine-api is present. this covers all our existing platforms and allows us to talk about future platforms that will be added to the machine-api without the need to keep a specific list.